attestor/command/cli.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2022 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package pkg defines fns for running Scorecard checks on a Repo.
+
+package command
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	// input flags
+	repoURL            string
+	commitSHA          string
+	mode               string
+	image              string
+	policyPath         string
+	attestationProject string
+	overwrite          bool
+	noteName           string
+	// input flags: pgp key flags
+	pgpPriKeyPath string
+	pgpPassphrase string
+	// pkix key flags
+	pkixPriKeyPath string
+	pkixAlg        string
+
+	// input flags: kms flags
+	kmsKeyName   string
+	kmsDigestAlg string
+)
+
+func addCheckFlags(cmd *cobra.Command) {
+	cmd.PersistentFlags().StringVar(&policyPath, "policy", "", "(required for check) scorecard attestation policy file path, e.g., /tmp/policy-binauthz.yml")
+	cmd.MarkPersistentFlagRequired("policy")
+	cmd.PersistentFlags().StringVar(&repoURL, "repo-url", "", "Repo URL from which source was built")
+	cmd.MarkPersistentFlagRequired("repo-url")
+	cmd.PersistentFlags().StringVar(&commitSHA, "commit", "", "Git SHA at which image was built")
+}
+
+func addSignFlags(cmd *cobra.Command) {
+	cmd.PersistentFlags().StringVar(&image, "image", "", "Image url, e.g., gcr.io/foo/bar@sha256:abcd")
+	cmd.MarkPersistentFlagRequired("image")
+	cmd.PersistentFlags().StringVar(&noteName, "note-name", "", "note name that created attestations are attached to, in the form of projects/[PROVIDER_ID]/notes/[NOTE_ID]")
```

Comment:
Is the `note-name` flag needed? I thought this was decided by the attestor right?

Reply:
Yes, previously it was configurable because that was how kritis did it, but I think making it a constant attestation name will allow binauthz to work more predictably